lib/core/src/server/mergeBabel.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-param-reassign,no-shadow */
```

Comment:
I've opened an issue - https://github.com/neutrinojs/babel-merge/issues/14
In any case, I will move it out of here later.